src/include/snapper/dialogs.rb

```diff
@@ -148,19 +138,19 @@ def ModifySnapshotPopup(snapshot)
       if type != :POST
         cont = VBox(
           # popup label, %{num} is number
-          Label(_("Modify Snapshot %{num}") % { :num => num }),
+          Label(format(_("Modify Snapshot %{num}"), num: num)),
```

Comment:
rubocop is not about better, it is about consistency. So it use one method for same operation. Ruby style guide suggest it https://github.com/bbatsov/ruby-style-guide#sprintf